yogstation/code/datums/ruins/station.dm

```diff
@@ -0,0 +1,8 @@
+/datum/map_template/ruin/station
+	prefix = "_maps/RandomRuins/StationRuins/"
+	cost = 0
```

Comment:
Careful that the ruinloader doesn't pick these up? Or have you accounted for that

Reply:
ruinloader does space ruins and lava ruins specifically, not ruins in general :)